deadspace/code/necromorph/bodyparts/e_exploder.dm

```diff
@@ -17,28 +18,59 @@
 	px_x = 0
 	px_y = -8
 	wound_resistance = 5
+	biomass = 9
 
-
-/obj/item/bodypart/leg/left/necromorph/exploder/enhanced
-	name = "yellow pustule"
+/obj/item/bodypart/arm/left/necromorph/exploder/enhanced
+	name = "red pustule"
 	limb_id = SPECIES_NECROMORPH_EXPLODER_ENHANCED
 	icon_static = 'deadspace/icons/necromorphs/exploder/exploder_enhanced.dmi'
-	icon_state = "l_leg"
+	icon_state = "l_arm"
 	attack_verb_continuous = list("kicks", "stomps")
 	attack_verb_simple = list("kick", "stomp")
-	max_damage = 50
+	max_damage = 75
 	px_x = -2
 	px_y = 12
 	wound_resistance = 0
+	biomass = 16 //The majority of biomass is in the arm, due to the much bigger explosion
 
-/obj/item/bodypart/leg/right/necromorph/exploder/enhanced
+/obj/item/bodypart/arm/right/necromorph/exploder/enhanced
 	name = "right arm"
 	limb_id = SPECIES_NECROMORPH_EXPLODER_ENHANCED
 	icon_static = 'deadspace/icons/necromorphs/exploder/exploder_enhanced.dmi'
-	icon_state = "l_leg"
+	icon_state = "r_arm"
 	attack_verb_continuous = list("kicks", "stomps")
 	attack_verb_simple = list("kick", "stomp")
-	max_damage = 50
+	max_damage = 75
 	px_x = 2
 	px_y = 12
 	wound_resistance = 0
+	biomass = 7
+
+/obj/item/bodypart/leg/right/necromorph/exploder/enhanced
+	name = "fused legs"
+	desc = "Two legs fused together to form a thick, meaty stalk."
+	plaintext_zone = "fused legs"
+	limb_id = SPECIES_NECROMORPH_EXPLODER_ENHANCED
+	bodypart_flags = BP_IS_MOVEMENT_LIMB | BP_NO_PAIN | STOCK_BP_FLAGS_ARMS & ~(BP_HAS_BONES|BP_HAS_TENDON|BP_HAS_ARTERY)
```

Comment:
Is this supposed to have `STOCK_BP_FLAGS_ARMS `?

Reply:
No they are not, knew I would miss something.